ldclient/__init__.py

```diff
@@ -11,6 +13,27 @@
                 "firstName", "lastName", "avatar", "name", "anonymous"]
 
 
+"""Settings."""
+client = None
+api_key = None
+start_wait = 5
+config = Config()
+
+_lock = threading.Lock()
+
+
+def get():
```

Comment:
This follows the model used by segment to enforce the singleton pattern. See updated readme + restwrapper PR: https://github.com/launchdarkly/python-restwrapper/pull/1/files